src/backend/Auth/Actions/ManageUsersActions/ManageUsersGetSingle/ManageUsersGetSingleAction.cs

```diff
@@ -38,6 +39,7 @@ public async Task<OkObjectResult> Execute(string id)
                  .Include(c => c.Roles)!
                      .ThenInclude(c => c.Role)
                  .Include(c => c.Warehouse)
+                 .Where(c => c.Id != UserSeeder.ServiceUser.Id)
```

Comment:
Już tu mogłeś sobie darować, bo czysto hipotetycznie nigdy się to nie stanie.
Ale może zostać

Reply:
fixed